xml/deployment_firstboot.xml

```diff
@@ -358,8 +358,8 @@
   <sect2 xml:id="sec.fb.customize.wf">
    <title>Customizing the Workflow</title>
    <para>
-    By default, a standard firstboot workflow includes the following
-    components:
+    The provided <filename>/etc/YaST2/firstboot.xml</filename> example, defines
```

Comment:
Remove the comma: ' The provided  [...] example defines [...]'.

Reply:
Agreed. Comma begone!

Reply:
Updated. Thanks!